app/controllers/PostsController.php

```diff
@@ -40,13 +40,13 @@ public function rssAction()
         $feed->setDescription($this->config->rss->description);
         $feed->setLink($this->getFullUrl());
 
-        $posts = $this->finder->getLatest($this->config->blog->postsPerPage);
+        $posts = $this->finder->getLatest(1);
```

Comment:
No it is page 1. The prototype of the `getLatest` changed to only return the page. The number of posts is controlled by the config file `$this->config->blog->postsPerPage`